x/gov/keeper/deposit.go

```diff
@@ -9,11 +9,12 @@ import (
 	sdkmath "cosmossdk.io/math"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	disttypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
 	"github.com/cosmos/cosmos-sdk/x/gov/types"
 	v1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
 )
 
+const poolModuleName = "protocol-pool"
```

Comment:
Given that we import pool anyway in gov, can we not just its name from the pool package?